client/dashboard/src/pages/toolsets/ToolSelect.tsx

```diff
@@ -24,7 +24,7 @@ import { useToolset, useUpdateToolsetMutation } from "@gram/client/react-query";
 import { useListTools } from "@gram/client/react-query/listTools.js";
```

Comment:
This change fixes a double write we were doing from the toolselect page. Any time a tool was toggled, we called updateToolset twice